gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -700,6 +822,11 @@ public Builder setEndpoint(String endpoint) {
       return this;
     }
 
+    public Builder setEndpointContext(EndpointContext endpointContext) {
+      this.endpointContext = endpointContext;
+      return this;
+    }
```

Comment:
Is this possible to be package-private?

Reply:
Yes, done in https://github.com/googleapis/sdk-platform-java/pull/3326/commits/f9eef5b0e0ef3f2df1ba659693d836f5ba547eb7